### ISSUE TITLE
[Accessibility]A11y_.NET Online REPL View_Color Contrast: The color contrast ratio for the line numbers in the editor is 3.7:1 which is < 4.5:1

### ISSUE THREAD

**Issue by soumyatippapuram** [dotnet/try#105]
**Environment Details:** 
URL: https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
Browser: Edge 42.17134.1.0

**Repro Steps:**
1)Hit the URL : https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
2)Tab till "Enter focus mode" button and press enter
3)Check the color contrast ratio for all the contents in the editor pane 

**Actual Result:**
The color contrast ratio for the line numbers in the editor is 3.7:1 which is < 4.5:1

**Expected Result:**
The color contrast ratio for the line numbers in the editor should be >= 4.5:1

**User Impact:** 
Visually impaired users will find it difficult to access the line numbers when errors are shown in the output section using line number

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
![bug_color contrast for numbers](https://user-images.githubusercontent.com/41108985/42629429-e45831fe-85f0-11e8-9f13-28c94fab120d.png)

**Comment by LadyNaggaga** [comment]
https://github.com/Microsoft/monaco-editor/issues/367
 The Monaco team is shipping with a High Contrast Theme
Please see https://github.com/Microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this is non blocking scenario.
